new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItDomainOnPV.java

```diff
@@ -0,0 +1,706 @@
+        .createNamespacedJob(jobBody), "Domain creation job failed");
+
+    logger.info("Checking if the domain creation job {0} completed in namespace {1}",
+        jobName, namespace);
+    withStandardRetryPolicy
+        .conditionEvaluationListener(
+            condition -> logger.info("Waiting for job {0} to be completed in namespace {1} "
+                + "(elapsed time {2} ms, remaining time {3} ms)",
+                jobName,
+                namespace,
+                condition.getElapsedTimeInMS(),
+                condition.getRemainingTimeInMS()))
+        .until(jobCompleted(jobName, null, namespace));
+  }
+
+  /**
+   * Create secret for docker credentials.
+   */
+  private void createOCRRepoSecret() {
+
+    // docker login, if necessary
+    logger.info("docker login to OCR registry");
+    assertTrue(dockerLogin(OCR_REGISTRY, OCR_USERNAME, OCR_PASSWORD), "login to OCR failed");
+
+
+    logger.info("Creating repository registry secret in namespace {0}", domainNamespace);
+    JsonObject dockerConfigJsonObject = createDockerConfigJson(
+        OCR_USERNAME, OCR_PASSWORD, OCR_EMAIL, OCR_REGISTRY);
+    String dockerConfigJson = dockerConfigJsonObject.toString();
+
+    // Create the V1Secret configuration
+    V1Secret repoSecret = new V1Secret()
+        .metadata(new V1ObjectMeta()
+            .name(OCR_SECRET_NAME)
+            .namespace(domainNamespace))
+        .type("kubernetes.io/dockerconfigjson")
+        .putDataItem(".dockerconfigjson", dockerConfigJson.getBytes());
+
+    boolean secretCreated = assertDoesNotThrow(() -> createSecret(repoSecret),
+        String.format("createSecret failed for %s", OCR_SECRET_NAME));
+    assertTrue(secretCreated, String.format("createSecret failed while creating secret %s", OCR_SECRET_NAME));
+  }
+
+  /**
+   * Create secret for WebLogic credentials.
+   */
+  private void createWebLogicCredentialsSecret() {
+    logger.info("Creating secret for WebLogic credentials");
+    wlSecretName = "weblogic-credentials";
+    Map<String, String> adminSecretMap = new HashMap<>();
+    adminSecretMap.put("username", adminUser);
+    adminSecretMap.put("password", adminPassword);
+    boolean secretCreated = assertDoesNotThrow(() -> createSecret(new V1Secret()
+        .metadata(new V1ObjectMeta()
+            .name(wlSecretName)
+            .namespace(domainNamespace))
+        .stringData(adminSecretMap)), "Create secret failed with ApiException");
+    assertTrue(secretCreated, String.format("create secret failed for %s", wlSecretName));
+  }
+
+  /**
+   * Create a persistent volume and persistent volume claim.
+   * @throws IOException when creating pv path fails
+   */
+  private void createPVandPVC() throws IOException {
+    logger.info("creating persistent volume and persistent volume claim");
+
+    Path pvHostPath = Files.createDirectories(Paths.get(
+        TestConstants.PV_ROOT, this.getClass().getSimpleName(), domainUid + "-persistentVolume"));
```

Comment:
you can static import TestConstants.PV_ROOT here.

Reply:
fixed